dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/AbstractConfigTest.java

```diff
@@ -1002,7 +1002,7 @@ void testMetaData() throws Exception {
         List<Class<? extends AbstractConfig>> configClasses = Arrays.asList(ApplicationConfig.class,
                 ConsumerConfig.class, ProviderConfig.class, ReferenceConfig.class, ServiceConfig.class,
                 ProtocolConfig.class, RegistryConfig.class, ConfigCenterConfig.class, MetadataReportConfig.class,
-                ModuleConfig.class, SslConfig.class, MetricsConfig.class, MonitorConfig.class, MethodConfig.class);
+                ModuleConfig.class, SslConfig.class, MonitorConfig.class, MethodConfig.class);
```

Comment:
Why change this?

Reply:
Is there any problem if set null as default?

Reply:
Fixed